src/lib/components/event/event-summary.svelte

```diff
@@ -77,30 +77,28 @@
   $: updating = currentEvents.length && !fullHistory.length;
 </script>
 
-{#key [$eventFilterSort, category]}
-  <Pagination
-    floatId="event-view-toggle"
-    {items}
-    {updating}
-    let:visibleItems
-    let:activeRowIndex
-    let:setActiveRowIndex
-    aria-label="recent events"
-  >
-    <EventSummaryTable {updating} {compact} on:expandAll={handleExpandChange}>
-      {#each visibleItems as event, index (`${event.id}-${event.timestamp}`)}
-        <EventSummaryRow
-          {event}
-          {compact}
-          {visibleItems}
-          expandAll={$expandAllEvents === 'true'}
-          {initialItem}
-          active={activeRowIndex === index}
-          onRowClick={() => setActiveRowIndex(index)}
-        />
-      {:else}
-        <EventEmptyRow {loading} />
-      {/each}
-    </EventSummaryTable>
-  </Pagination>
-{/key}
+<Pagination
```

Comment:
Looks like filtering by `Workflow Events` is not working as expected 😞 The first time a filter is selected it works, but any ones after are not updating events.

| List | Filter |
|--|--|
|![Screenshot 2023-03-27 at 10 33 56 AM](https://user-images.githubusercontent.com/15069288/228005882-45428141-0948-4b17-804b-073408b3f7d4.png)|![Screenshot 2023-03-27 at 10 34 11 AM](https://user-images.githubusercontent.com/15069288/228006050-885111e5-f025-42a4-9edb-2f3cdd3b3e31.png)|